html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/window-failure.https.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/common/get-host-info.sub.js"></script>
+<div id=log></div>
+<script>
+async_test(t => {
+  const frame = document.createElement("iframe");
+  t.add_cleanup(() => frame.remove());
+  frame.src = get_host_info().HTTPS_NOTSAMESITE_ORIGIN + new URL("resources/iframe-failure.html", location).pathname;
+  window.onmessage = t.unreached_func("Got a message event, expected a messageerror event");
```

Comment:
+frame.onerror = t.unreached_func(...);

Reply:
As that event is not defined for that element I'd rather not add that in case we want to add it in the future.